packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -51,6 +51,17 @@ abstract class ScrollPhysics {
     return parent.applyPhysicsToUserOffset(position, offset);
   }
 
+  /// Whether the scrollable should let the user adjust the scroll offset, for
+  /// example by dragging.
+  ///
+  /// By default, the user can manipulate the scroll offset if there is some
+  /// difference between the min and max scroll extent.
```

Comment:
This is certainly correct however it might be worth reiterating the point in layman's terms. Something like: [in other words], if it's not possible for the user to expose more of the scrollable by tapping or dragging, then return false.